server/src/main/java/org/apache/iotdb/db/mpp/plan/parser/ASTVisitor.java

```diff
@@ -1170,10 +1187,8 @@ private String parseStringLiteralInInsertValue(String src) {
   }
 
   private String parseIdentifier(String src) {
-    if (2 <= src.length() && src.charAt(0) == '`' && src.charAt(src.length() - 1) == '`') {
-      String unescaped = StringEscapeUtils.unescapeJava(src.substring(1, src.length() - 1));
-      // replace `` with `
-      return unescaped.replace("``", "`");
+    if (src.startsWith("`") && src.endsWith("`")) {
```

Comment:
Use a common constant instead of "`" and  "``" directly.

Reply:
fixed.